File: Php/app/Urls/PlayerInviteConfirmationUrlModel.php
namespace app\Urls{

	use app\RouteFormats;
	use app\Urls\BaseClasses\PlayerUrlModel;
	use entities\Player;
	use entities\Homegame;

	class PlayerInviteConfirmationUrlModel extends PlayerUrlModel{

		public function __construct(Homegame $homegame, Player $player){
			parent::__construct(RouteFormats::playerInviteConfirmation, $homegame, $player);
		}

	}

}

File: Web/Models/UrlModels/PlayerInviteConfirmationUrlModel.cs
using Core.Classes;
using Web.Routing;

namespace Web.Models.UrlModels{

	public class PlayerInviteConfirmationUrlModel : PlayerUrlModel{

		public PlayerInviteConfirmationUrlModel(Homegame homegame, Player player)
            : base(RouteFormats.PlayerInviteConfirmation, homegame, player)
        {
		}

	}

}